src/index.html

```diff
@@ -8,9 +8,40 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="post"
+      class="search-form"
+      data-qa="big"
+    >
+      <label>
+        <input
+        type="text"
+        class="search-city search-city-big"
+        name="searchCity"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
```

Comment:
Pay attention on indents
```suggestion
        <input
          type="text"
          class="search-city search-city-big"
          name="searchCity"
          data-qa="keypress"
          placeholder="Try &quot;Los Angeles&quot;"
        >
```